docs/build_version_doc/artifacts/0.12.0.index.html

```diff
@@ -0,0 +1,1241 @@
+<!DOCTYPE html>
```

Comment:
Is this intended?

Reply:
Yes, they are fine. It'll bring it up to what is currently on master, but will have no impact to a plain `make docs`. They're used when building the full site with all of the versions. These files get deleted with #11191 and the build script is updated to no longer use these static files.